Module not found: Can't resolve 'path' on NextJS `edge` runtime.

## Summary
When running langchainjs on the `edge` runtime in NextJS I see the following exception:
```
Error: Module not found: Can't resolve 'path'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/langchain/dist/base_language/count_tokens.js
./node_modules/langchain/dist/llms/openai.js
./node_modules/langchain/llms/openai.js
```

Currently running version `0.0.63`.

--- a/langchain/package.json
+++ b/langchain/package.json
@@ -467,7 +467,7 @@
   },
   "dependencies": {
     "@anthropic-ai/sdk": "^0.4.3",
-    "@dqbd/tiktoken": "^1.0.7-alpha.0",
+    "@dqbd/tiktoken": "^1.0.7",
     "binary-extensions": "^2.2.0",
     "browser-or-node": "^2.1.1",
     "expr-eval": "^2.0.2",

--- a/langchain/package.json
+++ b/langchain/package.json
@@ -467,7 +467,7 @@
   },
   "dependencies": {
     "@anthropic-ai/sdk": "^0.4.3",
-    "@dqbd/tiktoken": "^1.0.4",
+    "@dqbd/tiktoken": "^1.0.7-alpha.0",
     "binary-extensions": "^2.2.0",
     "browser-or-node": "^2.1.1",
     "expr-eval": "^2.0.2",

